src/modules/acbs/acbs-facility-party.service.ts

```diff
@@ -29,4 +32,21 @@ export class AcbsFacilityPartyService {
       }),
     });
   }
+
+  async getFacilityPartiesForFacility(portfolioIdentifier: string, facilityIdentifier: UkefId, idToken: string): Promise<AcbsGetFacilityPartyResponseDto[]> {
+    const { data: facilityPartiesInAcbs } = await this.acbsHttpService.get<AcbsGetFacilityPartyResponseDto[]>({
+      path: `/Portfolio/${portfolioIdentifier}/Facility/${facilityIdentifier}/FacilityParty`,
+      idToken,
+      onError: createWrapAcbsHttpPostErrorCallback({
+        messageForUnknownError: `Failed to get a party for facility ${facilityIdentifier} in ACBS.`,
+        knownErrors: [getFacilityNotFoundKnownAcbsError(facilityIdentifier)],
```

Comment:
Should we include this? This endpoint in ACBS doesn't return a `Facility not found or user does not have access` message so I was expecting `knownErrors` to be `[]`

Reply:
Thanks, was not sure how knownErrors works. Removed it and tests are still passing.